doc/plugins/datalogger/datalogger.rst

```diff
@@ -0,0 +1,137 @@
+.. image:: ../../resources/datalogger/datalogger1.png
+    :align: center
+
+Display Text
+---------------------------------------------------------------------
+
+Using the left side menu user can select from the available devices, for the selected device channels last read value and the unit of measurement will be displayed in text format
+
+To use this option click “Text” button on the bottom right
+
+.. image:: ../../resources/datalogger/datalogger2.png
+    :align: center
+
+
+Display Seven Segment
+---------------------------------------------------------------------
+
+Using the left side menu user can select from the available devices, for the selected device channels last read value, minimum and maximum recorded values and the unit of measurement will be displayed in seven segment display format
+
+To use this option click “7 Segment” button on the bottom right
+
+.. image:: ../../resources/datalogger/datalogger3.png
+    :align: center
+
+---------------------------------------------------------------------
+
+Settings
+--------------------------------------------------------------------
+
+
+The gear button in the top right corner of the plugin will open the settings allowing user to alter configurations as follow 
+
+	1. Changing the title in settings from “DataMonitor” will reflect in the tool name
+
+.. image:: ../../resources/datalogger/datalogger_settings_1.PNG
+	:scale: 80 %
+	:alt: alternate text
+	:align: right
+
+
+.. image:: ../../resources/datalogger/datalogger_settings_2.PNG
+	:scale: 80 %
+	:alt: alternate text
+	:align: right
+
+|
+
+	2. X-AXIS : settings for x axis 
+
+		2.1. Toggle between displaying real time or delta values (Use UTC date and time) . Real time values are based on the system time, delta values consider starting point as 0 and represents the amount of time passed since then.
+
+		2.2. Toggle “LIVE PLOTTING”
+			* when enabled will update the plot to the current time for each read
+			* when disabled will allow user to pick a date and a time to go to on the plot
+
+		2.3. Delta represents the interval displayed on the plot
+
+|
+
+.. image:: ../../resources/datalogger/datalogger_settings_4.PNG
+	:scale: 80 %
+	:alt: alternate text
+	:align: right
+
+|
+
+	3. Y-AXIS: settings for y axis 
+
+		3.1. Autoscale option changes the minimum and maximum of Y-Axis to fit all values recorded
+
+		3.2. Min and Max values will update the minimum respectively the maximum values of Y-Axis, this only works if **Autoscale** is off
+
+	4. Curve settings affect all curves on the plot user can set the curves thickness and style
+
+	5. Plot settins are used to show/hide plot items 
+
+	6. Settings for seven segment section allow user to set precision of display values and toggle on/off the minimum and maximum values
+
+	7. Data logging section is used for saving and importing data from and to a specified CSV file like the one bellow
+
+|
+
+.. image:: ../../resources/datalogger/datalogger_settings_5.PNG
+	:align: center
+
+
```

Comment:
Example of using spaces and 80 column
```suggestion
Settings
--------------------------------------------------------------------------------

The gear button in the top right corner of the plugin will open the settings
allowing user to alter configurations as follow

1. Changing the title in settings from “DataMonitor” will reflect in the tool
   name

.. image:: ../../resources/datalogger/datalogger_settings_1.PNG
        :scale: 80 %
        :alt: alternate text
        :align: right


.. image:: ../../resources/datalogger/datalogger_settings_2.PNG
        :scale: 80 %
        :alt: alternate text
        :align: right

2. X-AXIS : settings for x axis

   2.1. Toggle between displaying real time or delta values (Use UTC date and
        time).
        Real time values are based on the system time, delta values consider
        starting point as 0 and represents the amount of time passed since then.
   2.2. Toggle “LIVE PLOTTING”
        * when enabled will update the plot to the current time for each read
        * when disabled will allow user to pick a date and a time to go to on
          the plot

   2.3. Delta represents the interval displayed on the plot

.. image:: ../../resources/datalogger/datalogger_settings_4.PNG
        :scale: 80 %
        :alt: alternate text
        :align: right

3. Y-AXIS: settings for y axis

   3.1. Autoscale option changes the minimum and maximum of Y-Axis to fit all
        values recorded

   3.2. Min and Max values will update the minimum respectively the maximum
        values of Y-Axis, this only works if **Autoscale** is off

4. Curve settings affect all curves on the plot user can set the curves
   thickness and style

5. Plot settins are used to show/hide plot items

6. Settings for seven segment section allow user to set precision of display
   values and toggle on/off the minimum and maximum values

7. Data logging section is used for saving and importing data from and to a
   specified CSV file like the one bellow

.. image:: ../../resources/datalogger/datalogger_settings_5.PNG
        :align: center

```